matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/internal/DefaultPhenotypesMap.java

```diff
@@ -219,17 +219,29 @@ public static void reorder(List<Map<String, String>> predefined1, List<Map<Strin
             if (StringUtils.equals(name1, name2)) {
                 common1.add(item1);
```

Comment:
this is for the if statement on line 237 (in the new code) and cycle on line 245 - if `item1` is `null` at that point, the cycle shouldn't start